deploy/charts/cert-manager/README.md

```diff
@@ -90,7 +90,7 @@ The following table lists the configurable parameters of the cert-manager chart
 | `global.podSecurityPolicy.useAppArmor` | If `true`, use Apparmor seccomp profile in PSP | `true` |
 | `global.leaderElection.namespace` | Override the namespace used to store the ConfigMap for leader election | `kube-system` |
 | `image.repository` | Image repository | `quay.io/jetstack/cert-manager-controller` |
-| `image.tag` | Image tag | `v0.13.0` |
+| `image.tag` | Image tag | `v0.14.0` |
```

Comment:
`v0.14.0-alpha.0` will be the tag that we want to be visible on `hub.helm.sh`

Reply:
Changed!